src/functionsShellCommandAction.js

```diff
@@ -9,17 +9,20 @@ var util = require("util");
 var serveFunctions = require("./serve/functions");
 var LocalFunction = require("./localFunction");
 var logger = require("./logger");
+var shell = require("./emulator/functionsEmulatorShell");
 
 module.exports = function(options) {
   options.port = parseInt(options.port, 10);
 
   return serveFunctions
-    .start(options)
+    .start(options, { quiet: true })
```

Comment:
Should we maybe change `quiet` to be a verbosity level like `"QUIET" | "NORMAL" | "VERBOSE"` so that we could replace `_is_verbose` in tests and just allow us to set a level on `EmulatorLogger`?

Reply:
Couldn't quite get there (JS/TS fun) but introduced the concept of verbosity.

Reply:
So you can set that in tests if you want.